vue-components/src/components/Button.vue

```diff
@@ -23,11 +26,12 @@ Vue.use( VueCompositionAPI );
  *
  * Known issues:
  *
- * * The styles defined on `:focus` do not apply in Safari and Firefox on macOS.<br>
- * This seems to be a desired behavior and not a bug<br>
- * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus<br>
+ * * The styles defined on `:focus` do not apply in Safari and Firefox on macOS.
```

Comment:
I think something went wrong here when resolving the conflicts or merging master in. These trailing `<br>` should not be deleted. They were just added in #331 . Conversly, the `/* eslint-enable no-trailing-spaces */` should not be here anymore.